runtime-modules/content/src/tests/transfers.rs

```diff
@@ -8,78 +8,333 @@ use sp_core::sp_std::iter::FromIterator;
+        increase_account_balance_helper(
+            ContentTreasury::<Test>::account_for_channel(ChannelId::one()),
+            DEFAULT_PAYOUT_EARNED
+                // TODO: Should be changed to bloat_bond after https://github.com/Joystream/joystream/issues/3511
+                .saturating_add(ed.into()),
+        );
+        IssueCreatorTokenFixture::default().call_and_assert(Ok(()));
+        IssueRevenueSplitFixture::default()
+            .with_starting_block(SPLIT_STARTING_BLOCK)
+            .call_and_assert(Ok(()));
+
+        run_to_block(SPLIT_STARTING_BLOCK + 1);
 
-        create_initial_storage_buckets_helper();
-        increase_account_balance_helper(DEFAULT_MEMBER_ACCOUNT_ID, INITIAL_BALANCE);
-        create_default_member_owned_channel();
+        InitializeChannelTransferFixture::default()
+            .with_new_member_channel_owner(THIRD_MEMBER_ID)
+            .call_and_assert(Err(
+                Error::<Test>::ChannelTransfersBlockedDuringRevenueSplits.into(),
+            ));
+    })
+}
 
+#[test]
+fn initialize_channel_transfer_fails_during_unfinalized_revenue_split() {
+    pub const SPLIT_STARTING_BLOCK: u64 = 10;
+    with_default_mock_builder(|| {
+        let ed = <Test as balances::Config>::ExistentialDeposit::get();
+        ContentTest::with_member_channel().setup();
+        increase_account_balance_helper(
+            ContentTreasury::<Test>::account_for_channel(ChannelId::one()),
+            DEFAULT_PAYOUT_EARNED
+                // TODO: Should be changed to bloat_bond after https://github.com/Joystream/joystream/issues/3511
+                .saturating_add(ed.into()),
+        );
+        IssueCreatorTokenFixture::default().call_and_assert(Ok(()));
+        IssueRevenueSplitFixture::default()
+            .with_starting_block(SPLIT_STARTING_BLOCK)
+            .call_and_assert(Ok(()));
+
+        run_to_block(SPLIT_STARTING_BLOCK + DEFAULT_REVENUE_SPLIT_DURATION + 1);
+
+        InitializeChannelTransferFixture::default()
+            .with_new_member_channel_owner(THIRD_MEMBER_ID)
+            .call_and_assert(Err(
+                Error::<Test>::ChannelTransfersBlockedDuringRevenueSplits.into(),
+            ));
+    })
+}
+
+#[test]
+fn initialize_channel_transfer_fails_during_upcoming_token_sales() {
+    pub const SALE_STARTING_BLOCK: u64 = 10;
+    with_default_mock_builder(|| {
+        ContentTest::with_member_channel().setup();
+        IssueCreatorTokenFixture::default().call_and_assert(Ok(()));
+        InitCreatorTokenSaleFixture::default()
+            .with_start_block(SALE_STARTING_BLOCK)
+            .call_and_assert(Ok(()));
+        InitializeChannelTransferFixture::default()
+            .with_new_member_channel_owner(THIRD_MEMBER_ID)
+            .call_and_assert(Err(
+                Error::<Test>::ChannelTransfersBlockedDuringTokenSales.into()
+            ));
+    })
+}
+
+#[test]
+fn initialize_channel_transfer_fails_with_lead_and_non_curator_channel() {
```

Comment:
Test cases like:
- `initialize_channel_transfer_fails_with_lead_and_non_curator_channel`
- `initialize_channel_transfer_fails_with_curator_non_authorized_channel`
- `initialize_channel_transfer_fails_with_member_actor`
- `initialize_channel_transfer_fails_with_non_channel_owner`

Will already be covered by:
- `unsuccessful_curator_channel_actions_with_invalid_context`
- `unsuccessful_member_channel_actions_with_invalid_context`

I added those to normalize invalid-context tests across all related content extrinsics, since creating them manually was a very repetitive and error-prone task